command_line/find_bad_pixels.py

```diff
@@ -40,48 +45,9 @@
 )
 
 
-def run(args):
-
-    from dials.util.options import OptionParser
-    from dials.util.options import flatten_experiments
-    from dials.util.command_line import ProgressBar
-
-    usage = "%s [options] data_master.h5" % (libtbx.env.dispatcher_name)
-
-    parser = OptionParser(
-        usage=usage,
-        phil=phil_scope,
-        read_experiments=True,
-        read_experiments_from_images=True,
-        epilog=help_message,
-    )
-
-    params, options = parser.parse_args(show_diff_phil=True)
-
-    experiments = flatten_experiments(params.input.experiments)
-    if len(experiments) != 1:
-        parser.print_help()
-        print("Please pass an experiment list\n")
-        return
-
-    imagesets = experiments.imagesets()
-
-    if len(imagesets) != 1:
-        raise Sorry("Please pass an experiment list that contains one imageset")
-
-    imageset = imagesets[0]
-
-    first, last = imageset.get_scan().get_image_range()
-    images = range(first, last + 1)
-
-    if params.images is None and params.image_range is not None:
-        start, end = params.image_range
-        params.images = list(range(start, end + 1))
-
-    if params.images:
-        if min(params.images) < first or max(params.images) > last:
-            raise Sorry("image outside of scan range")
-        images = params.images
+def find_constant_signal_pixels(imageset, images):
+    """Find pixels which are constantly reporting as signal through the
+    images in imageset."""
```

Comment:
This might be a good place to include a brief description of the algorithm you are using to find the bad pixels.

Reply:
True - will pop that into the TODO stack